package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@leukeleu/prettier-config",
   "repository": "https://github.com/leukeleu/prettier-config.git",
-  "version": "2.1.0",
+  "version": "3.0.0",
```

Comment:
We bump the version in github actions (which will also create a tag for release), so please undo this (see also: https://github.com/leukeleu/prettier-config/blob/main/CONTRIBUTORS.md#releasing-this-package)

Reply:
Was not aware of this workflow, will read up on it!